openmdao/core/system.py

```diff
@@ -3609,7 +3609,7 @@ def _update_response_meta(self, meta, get_size=False, use_prom_ivc=False):
             key = src_name
 
         meta['source'] = src_name
-        meta['distributed'] = \
+        meta['distributed'] = dist = \
```

Comment:
dist is never used

Reply:
it's used on line 3629 as part of the `if`